src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -196,33 +196,52 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
     }
 
     private setEditableFormFields(): void {
+        this.editFormFields.set('communicatiekanaal',
+            new SelectFormFieldBuilder().id('communicatiekanaal').label('communicatiekanaal')
+                                        .value({
```

Comment:
Kun je hier niet gewoon this.zaak.communicatiekanaal meegeven? Als het goed is geef je met optionLabel aan welk veld binnen het object getoond moet worden als label per option in de lijst. (zie ook hoe de edit-autocomplete werkt)